src/mocks/react-native-toast-message.mock.js

```diff
@@ -0,0 +1,6 @@
+export const mockShowToast = jest.fn();
+
+jest.mock('react-native-toast-message', () => ({
+  ...jest.requireActual('react-native-toast-message'),
```

Comment:
use `jest.requireActual` only when it's required to import the whole module.
create `mockModule` variable instead of creating one for each property